### PR TITLE
feat: new icon for development flags

### LINKED ISSUE #1225: Add feature flag menu to sidebar on localhost
### Overview

We have a feature flag menu that can be accessed via a keyboard shortcut `alt+shift+F` (Windows/Linux) or `opt+shift+F` (macOS).

Developers frequently want to use this locally and the shortcut is useful for operations and product to use in hosted environments, but is annoying for developers locally, and we don't want this menu showing to users in production.

### Reference

* Add it here https://github.com/shapeshift/web/blob/develop/src/Routes/Routes.tsx#L148

### Acceptance Criteria

* Name it `Feature Flags` and use a flag icon from font awesome
* The menu should only appear for `localhost`, never in another environment.

### Need By Date

_No response_

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by lolboysg**
## Description

- I added a flag icon in the navigation bar to go to the "/flags" route
- It does show on localhost
- It doesn't show in production

## Notice

- [X] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [X] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #1225 

## Risk

Not risky as it just adds a UI feature

## Testing

- Go to main page on localhost
- Click on the flag icon in the navigation bar

## Screenshots (if applicable)

image

### CODE REVIEW

File: src/Routes/Routes.tsx
Comment (NeOMakinG) [inline src/Routes/Routes.tsx:162]: It's fine until you are using the app locally without any docker or vhost, but using a `.env` variable would be way more robust

Maybe webpack could provide the `production` or `development` value too

Wait for a maintainer advice before applying my review please 👍

Comment (0xdef1cafe) [reply src/Routes/Routes.tsx:162]: @lolboysg try using `getConfig().isDev()` here as a more robust check

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**
